rcp/org.testeditor.rcp4.views.tcltestrun/src/org/testeditor/rcp4/views/tcltestrun/console/TestExecutionLogViewPart.xtend

```diff
@@ -26,7 +26,9 @@ class TestExecutionLogViewPart {
 	@PostConstruct
 	def void buildUI(Composite parent) {
 		browser = new Browser(parent, SWT.NORMAL)
-		browser.setUrl("http://localhost:19091/testlogs/web-app/index.html")
+		val url = "http://localhost:" + System.getProperty("org.osgi.service.http.port") +
```

Comment:
Should we maybe allow to set a specific port as well?
We could check a Test-Editor system property first and if it's not set use the org.osgi.service.http.port.
Also I'd prefer assigning the port to a local variable and use the `val url = '''http://localhost:«port»/testlogs/web-app/index.html'''` syntax.